### DISCUSSION TITLE
Tailscale image - podman build fails

### DISCUSSION THREAD

**Discussion by cmonty14** [containers/podman#21315] [Q&A]
Hi,
I'm trying to deploy [Tailscale docker image](https://hub.docker.com/r/tailscale/tailscale) on Ubuntu 22.04 using `podman`.
The instructions are simple and clear:
1. `docker pull tailscale/tailscale:stable`
2. `docker build -t tailscale/tailscale .`
3. `docker run -d --name=tailscaled -v /var/lib:/var/lib -v /dev/net/tun:/dev/net/tun --network=host --cap-add=NET_ADMIN --cap-add=NET_RAW tailscale/tailscale`

Here the first issue occurs:
`Error: stat /home/thomas/Dockerfile: no such file or directory`

However I can run `docker run -d --name=tailscaled -v /var/lib:/var/lib -v /dev/net/tun:/dev/net/tun --network=host --cap-add=NET_ADMIN --cap-add=NET_RAW tailscale/tailscale` and the container is available with `podman ps -a`.
```
CONTAINER ID  IMAGE                                 COMMAND               CREATED         STATUS                     PORTS       NAMES
3d73bf91169f  docker.io/tailscale/tailscale:latest  /usr/local/bin/co...  24 minutes ago  Exited (1) 23 minutes ago              tailscaled
```

Can you please advise what's wrong here?
Is `docker build¸ required?

THX

**Comment by rhatdan** [comment]
If you pull the image then probably no reason to build. 

Error: stat /home/thomas/Dockerfile: no such file or directory
means you don't have a Dockerfile or Containerfile to build with, so do not need to build.

Also are you using alias docker=podman or actually using podman.

Does the podman run command work, then you are all set.

**Comment by cmonty14** [comment]
I'm using `alias docker=podman`.
